posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Deploy Java applications to IBM Cloud Code Engine using the Paketo Buildpack for Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars.githubusercontent.com/u/29545669?v=4
+author_github: https://github.com/kevin-ortega
+seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
+seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
+blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
+open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
+open-graph-image-alt: screen capture of Code Engine build details
+---
+= Deploy Java applications to IBM Cloud Code Engine using the Paketo Buildpack for Liberty
+Kevin Ortega <https://github.com/kevin-ortega>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+You can use link:https://www.ibm.com/cloud/code-engine[IBM Cloud Code Engine] to easily deploy Java applications by using the link:https://openliberty.io/blog/2022/04/01/cloud-native-liberty-buildpack.html[Paketo Buildpack for Liberty] with your container images or source code.
+
+This post provides instructions on how to use the Paketo Buildpack for Liberty with Code Engine. To learn more about using Code Engine, see link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-getting-started[Getting started with IBM Cloud Code Engine].
+
+With the Paketo Buildpack for Liberty, you can use Open Liberty or WebSphere Liberty to run your application.
+
+IBM Cloud announced the link:https://cloud.ibm.com/docs/cloud-foundry-public?topic=cloud-foundry-public-deprecation[deprecation] of Cloud Foundry.
+Code Engine provides the next level of easy application deployment and mature management of critical business applications in IBM Cloud.  Move your applications from the Cloud Foundry liberty-for-java buildpack to a cloud native solution using Code Engine and the Paketo Buildpack for Liberty.
+
+== What do you need?
+* An https://www.ibm.com/cloud[IBM Cloud] Account
+* Your application source
+
+To deploy container images, you'll also need:
+
+* The link:https://buildpacks.io/docs/tools/pack/[pack CLI]
+* https://www.docker.com/[Docker]
+* An externally accessible container registry that can host your images.
+
+For details on deploying your application from repository source code from the Code Engine console, see  link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code[Deploying your app from repository source code from the console].
+
+For details on deploying your application from repository source code with the CLI, see  link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code-cli[Deploying your app from repository source code with the CLI].
```

Comment:
The links of these sentences are redundant with the part before them (and also there's an extra space after 'see').  This should be restructured to remove the redundancy. perhaps something like 
For details on deploying your application from repository source code using:
* link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code[Code Engine console]
* link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code-cli[CLI]